packages/flutter_tools/lib/src/build_runner/web_fs.dart

```diff
@@ -297,6 +297,11 @@ class WebFs {
 class BuildDaemonCreator {
   const BuildDaemonCreator();
 
+  // TODO(jonahwilliams): find a way to get build checks working for flutter for web.
+  static const String _ignoredLine1 = 'Warning: Interpreting this as package URI';
+  static const String _ignoredLine2 = 'build_script.dart was not found in the asset graph, incremental builds will not work';
```

Comment:
cc @jakemac53 - I wonder if we should suppress this line ourselves if the `--skip-build-script-check` flag is used?

Reply:
Ya, that seems like it would make sense.